src/sentry/static/sentry/app/components/events/interfaces/request.jsx

```diff
@@ -49,21 +49,24 @@ class RequestInterface extends React.Component {
     let view = this.state.view;
 
     let fullUrl = data.url;
-    if (data.query) {
+    if (data.query && fullUrl) {
       fullUrl = fullUrl + '?' + data.query;
     }
-    if (data.fragment) {
+    if (data.fragment && fullUrl) {
       fullUrl = fullUrl + '#' + data.fragment;
     }
 
-    // lol
-    let parsedUrl = document.createElement('a');
-    parsedUrl.href = fullUrl;
+    let parsedUrl = null;
+    if (fullUrl) {
+      // lol
```

Comment:
Might want to say that you're leveraging the element's URL parsing behavior to extract URL components.